env.source.sample

```diff
@@ -37,6 +37,18 @@ token \
 here \
 "
 
+### OPS_UTILS_DIR
+### The OPS_UTILS_DIR setting is an absolute path to any necessary scripts you wish
+### to have automatically mounted into your container. This is mounted in the
+### `/root/sop-utils` directory in the container, and you can optionally add that to
+### the path below if you wish
+# export OPS_UTILS_DIR=/Users/myuser/path/to/ops-sop/v4/utils
+
+### OPS_UTILS_DIR_RW is a boolean flag to allow mounting the ops-sop directory as
+### read/write instead of the default readonly. This is useful when debugging or 
```

Comment:
this could also be a security risk as it allows escaping outside the container

Reply:
I think that given:

1. These containers are not long-lived processes on servers
2. The container defaults to read-only unless explicitly set otherwise

That this is a fair risk.

Reply:
K